.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+	// See http://go.microsoft.com/fwlink/?LinkId=827846
+	// for the documentation about the extensions.json format
```

Comment:
https://stackoverflow.com/a/4183018/2067574

Reply:
https://code.visualstudio.com/docs/languages/json#_json-with-comments

Reply:
but github doesn't know about VS format, and while reading red lines it's hard to understand, where an actual error is and where a github warning is

Reply:
approved, because these comments are auto-generated